lms/envs/common.py

```diff
@@ -2409,7 +2409,8 @@ def _make_locale_paths(settings):
     'multiselectfield',
 
     # Third party action planner xblock goals tracking application
-    'action_planner.actionplan_manager'
+    'action_planner.actionplan_manager',
+    'msp_assessment.msp_dashboard',
```

Comment:
It looks like under the heading of `# Third party action planner xblock goals tracking application`. How about adding a separate heading for it or just adding separator.